### PR TITLE
F/76 nomad 0.10

### PR DESCRIPTION
**Pull request by MatthiasScholz**
Releates to issue #76

### CODE REVIEW

**Comment by MatthiasScholz** [comment]
Verification done as in issue #81 described.

**Comment by MatthiasScholz** [comment]
# New Version

After merging the changes into the master a new minor version should be released.

File: test/Makefile [hunk 1]
```diff
@@ -68,15 +71,20 @@ os:
...
+	$(call echoinfo, "--- List available versions of major dependencies ---")
+	@go list -m -versions ${dependency_terratest}
+	@go list -m -versions ${dependency_nomad}
 
 dependencies_upgrade:
-	go get ${dependency_terratest}@${TT_VERSION}
+	go get ${dependency_terratest}@${dependency_terratest_version}
+	go get ${dependency_nomad}@${dependency_nomad_version}
 
 dependencies_cleanup:
```
[the remark this answers is not in the document]
Comment (MatthiasScholz) [reply test/Makefile]: tflint was removed with commit c1f4b0c.

File: modules/ami2/nomad-consul-docker-ecr-cni.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+  "variables": {
+    "nomad_version": "0.10.2",
+    "cni_version": "0.8.2",
+    "consul_module_version": "v0.7.3",
+    "consul_version": "1.6.2",
+    "aws_account_ids": ""
+  },
+  "builders": [
+    {
+      "ami_name": "nomad-consul-docker-ecr-amazon-linux2-{{isotime | clean_ami_name}}",
```
[the remark this answers is not in the document]
Comment (MatthiasScholz) [reply modules/ami2/nomad-consul-docker-ecr-cni.json]: The missing information was added with the commit d7fadee.

**Comment by ThomasObenaus** [comment]
PR looks good and can be merged.

**Pull request merged**